Lesson 1

Elena-Kozlovskaya opened the pull request.

Elena-Kozlovskaya committed:
Added homework lesson_1

Elena-Kozlovskaya committed:
Added overload method getSquare for class Square

usharik commented on src/main/java/com/kozlovskaya/java/interview/homework/lesson_1/task_1/Person.java:
```diff
@@ -0,0 +1,110 @@
+    private String address;
+    private String phone;
+    private int age;
+    private String gender;
```
При создании билдера имело бы смысл сделать конструктор приватным. Возможность создать экземпляр класса должна быть только через билдер

usharik commented on src/main/java/com/kozlovskaya/java/interview/homework/lesson_1/task_3/Figure.java:
```diff
@@ -0,0 +1,15 @@
+public abstract class Figure {
+    private String name;
+
+    abstract void getSquare();
```
Тут точно package-private доступ?

usharik commented on src/main/java/com/kozlovskaya/java/interview/homework/lesson_1/task_3/Square.java:
```diff
@@ -0,0 +1,24 @@
+        System.out.println("Площадь квадрата = " + result);
+    }
+
+    void getSquare(double a, double b){
```
Почему package-private

Elena-Kozlovskaya committed:
Corrected remarks:
1. Figure class - private to public String name;
2. Person class - public to private in NoArgs Constructor
